src/Controller/UserController.php

```diff
@@ -396,4 +398,81 @@ public function userChangePassword(
         $userRepository->save($userChangePassword);
         return new JsonResponse(['message' => 'Password successfully changed!'], Response::HTTP_OK);
     }
+
+    /**
+     * Get User List.
+     * @Rest\Get("/list")
+     * @SWG\Get(
+     *     tags={"User"},
+     *     summary="Get a list of all users",
+     *     description="Get a list of all users",
+     *     operationId="getUsers",
+     *     produces={"application/json"},
+     *     @SWG\Parameter(
+     *     description="Filtration by technologies",
+     *     in="query",
+     *     name="filter[technology][]",
+     *     required=false,
+     *     type="integer",
+     *     ),
+     *     @SWG\Parameter(
+     *     description="Sorting by seniority (asc or desc). Desc by default.",
+     *     in="query",
+     *     name="sortBy[seniority]",
+     *     required=false,
+     *     type="integer",
+     *     )
+     * )
+     * @SWG\Response(
+     *     response=200,
+     *     description="Successfull operation!",
+     *     @SWG\Schema(
+     *     type="array",
+     *     @Model(type=User::class, groups={"UserList"})
+     * )
+     * )
+     * @SWG\Response(
+     *     response=401,
+     *     description="Unauthorized.",
+     *     @SWG\Schema(
+     *     @SWG\Property(property="code", type="integer", example=401),
+     *     @SWG\Property(property="message", type="string", example="JWT Token not found"),
+     *     )
+     * )
+     * @param Request $request
+     * @param UserListFilter $userListFilter
+     * @param UserListSort $userListSort
+     * @param UserRepository $userRepository
+     * @return JsonResponse
+     */
+
+    public function getUserList(
+        Request $request,
+        UserListFilter $userListFilter,
+        UserListSort $userListSort,
+        UserRepository $userRepository
+    ): JsonResponse {
+
+        $filter = $request->query->get('filter');
+        $userListFilter->setFilterFields((array)$filter);
+
+        $sorting = $request->query->get('sortBy');
+        $userListSort->setSortingFields((array)$sorting);
```

Comment:
Pagination is missing. In order to have the possibility to extract all the items in one page we should handle the case when `per_page=-1`. If it is exactly `-1` => return all the items.